### PR TITLE
Update certificate-connector-install.md

### PR DESCRIPTION
**Pull request by byteben**
On the *Azure AD Sign In* page, the user account specified must have an Intune licence assigned too or sign-in will fail

### CODE REVIEW

**Comment by PRMerger14** [comment]
@byteben : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Brenduns** [comment]
#sign-off

File: memdocs/intune/protect/certificate-connector-install.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Use the following procedure to both configure a new connector and modify a previ
 
 5. On the *Prerequisites* page, the wizard runs several checks on the server before the configuration can begin. Review and resolve any errors or warnings before you continue.
 
-6. On the *Azure AD Sign In* page, select the environment that hosts your Azure Active Directory, and then select **Sign In**. You’ll then be asked to authenticate your access.
+6. On the *Azure AD Sign In* page, select the environment that hosts your Azure Active Directory, and then select **Sign In**. You’ll then be asked to authenticate your access. This user account must have an Intune licence assigned.
```
Comment (ShannonLeavitt) [inline memdocs/intune/protect/certificate-connector-install.md:104]:
```suggestion
6. On the *Azure AD Sign In* page, select the environment that hosts your Azure Active Directory, and then select **Sign In**. You’ll then be asked to authenticate your access. This user account must have an Intune license assigned.
```
Fixing typo

**Comment by PRMerger15** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**